crowd_anki/importer/import_dialog.py

```diff
@@ -149,10 +136,12 @@ def set_checked_and_text(checkbox, text, count, checked: bool = True):
                 text = f"{text}: {'{:,}'.format(count)}"
             checkbox.setText(text)
 
-        set_checked_and_text(self.form.cb_headers, "Headers", None)
+        self.form.cb_headers.setVisible(False)  # TODO: implement header selection functionality. Disabled for now
+        # set_checked_and_text(self.form.cb_headers, "Headers", None)
```

Comment:
just remove it for now ;)

Reply:
:sob: my babies :sob: okay done :+1: :grin: